docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -92,6 +98,30 @@ exclude_paths:
   - "**/*.resource"
 ```
 
+## Including specific files using a Codacy configuration file {: id="include-files"}
+
+The Codacy configuration file allows you to explicitly specify files or directories to include in the analysis. This is particularly useful for [bypassing files or directories that are ignored by default](./ignoring-files.md#default-ignored-files) or specified in `exclude_paths`.
```

Comment:
Maybe we can add a note/tip in the [Default ignored files](https://github.com/codacy/docs/blob/DOCS-699-document-new-field-in-codacy-config-file-include-paths/docs/repositories-configure/ignoring-files.md?plain=1#L29) section about this bypassing possibility? 🤔

Reply:
Good idea. Done 👍